### ISSUE TITLE
Documentation update list

### ISSUE THREAD

**Issue by smeech** [espanso/website#33]
I put a list of documentation updates on the main [repo Issue list](https://github.com/espanso/espanso/issues/1788), but it might fit better here.

**Comment by traptw1thin** [comment]
Another update I would propose is for https://espanso.org/docs/sync/. One, I don't think that Google Drive works for Linux due to the file system not showing the names of files but only their ids. But two, and I'm not sure if this is only on Linux (and Dropbox) but I think it is erroneous to say that you want to move the $CONFIG folder into the newly created espanso folder within the place you want to sync from. More accurate would be to say to move the _contents_ of the $CONFIG folder into this newly created folder. Then delete the $CONFIG folder and then modify the command (at least for linux) to `ln -s "/home/user/Dropbox/espanso" "/home/user/.config"` basically replacing the $CONFIG folder. Otherwise when you restart espanso you end up with the file paths "/home/user/.config/espanso/" and "/home/user/.config/espanso/espanso/" and inside the first one when you restart espanso you get another match and config folder resulting in another set of base.yml and everything else that would be very confusing if not non-functional if there were contradictions. Additionally there should be a mention that you need to run `espanso stop` before starting this or else very odd things will happen when you move things out of the $CONFIG folder.

**Comment by smeech** [comment]
That makes sense.
To shorten the instructions, I think you are saying:
- Determine $CONFIG result of `espanso path config`
- `espanso stop`
- Create a new espanso folder in the cloud storage
- Copy or move the contents of the $CONFIG folder into it
- Link command "new folder" to $CONFIG
- `espanso start`

Is that correct?

**Comment by traptw1thin** [comment]
That MIGHT work, I don't know because thats not quite what I did or meant, but it may be equivalent. I will explain the slight difference in case it does make a difference. 

The second to last bullet you listed is not quite what I was trying to describe. The $CONFIG that you get in the first bullet from `espanso path config` will be something like `/home/user/.config/espanso` however that second to last bullet would need to be:

- Link command "new espanso folder" to `/home/user/.config` (in Linux obviously diff for diff OS, just trying to express the difference)

The difference is per what you wrote above the second to last bullet's command would expand $CONFIG to `/home/user/.config/espanso`. Again, possibly what you wrote would also work but I think it would result in the double espanso/espanso directory structure still.

**Comment by traptw1thin** [comment]
Additionally in just attempting to get this sync setup on a Windows PC I also discovered the order of commands is backwards from what is on that page. In windows with mklink the order is `<link location> <target>` but that page of the documentation lists it as `<target> <link location>`

**Comment by smeech** [comment]
Would you refresh your view of docs, please, and recheck? I did make a change in the Windows `mklink` order recently, so I _think_ it should now be correct.

**Comment by smeech** [comment]
> The difference is per what you wrote above the second to last bullet's command would expand $CONFIG to `/home/user/.config/espanso`. Again, possibly what you wrote would also work but I think it would result in the double espanso/espanso directory structure still.

I'm not sure if this is correct. I tried the process here to test it out, and ran the command:
 `ln -s "/home/stephen/Cloud/Dropbox/espanso" /home/stephen/.config/espanso`
 which is what the docs suggest, and it worked as I expected.

**Comment by traptw1thin** [comment]
When I do that I end up with the link located at /home/user/.config/espanso/espanso, which I don't think is correct. This results in the following folder structure: 
![image](https://github.com/espanso/website/assets/51544299/edbcb42d-ec17-4900-93fa-836e34fbba39)
But if I delete the espanso folder at /home/user/.config and then run `ln -s "/home/charity/Dropbox/espanso/" "/home/charity/.config"` I get the following folder structure, which I thought was the goal: 
![image](https://github.com/espanso/website/assets/51544299/f9db4a02-62d1-45b7-9e69-483b409a45bf)

Maybe its a misunderstand on my part, but with the first folder structure there ends up being 2 `default.yml` files, 2 `packages` folders, and 2 `base.yml` files. One set at `/home/user/.config/espanso/config/default.yml`, `/home/user/.config/espanso/match/packages/`, and `/home/user/.config/espanso/match/base.yml` and the other set at `/home/user/.config/espanso/espanso/config/default.yml`, `/home/user/.config/espanso/espanso/match/packages/`, and `/home/user/.config/espanso/espanso/match/base.yml`. If the intent is to have two sets then maybe that's fine.

**Comment by smeech** [comment]
There should only be one set of configuration files.
I think the trick is perhaps to ensure the `/espanso` folder doesn't exist in `/home/user/.config` before running the link command?

**Comment by smeech** [comment]
New issue started #38 for further conversation.
